src/databricks/labs/ucx/source_code/queries.py

```diff
@@ -42,17 +49,22 @@ def lint(self, code: str) -> Iterable[Advice]:
                     end_col=1024,
                 )
 
-    @staticmethod
-    def _catalog(table):
+    def _catalog(self, table):
         if table.catalog:
             return table.catalog
-        return 'hive_metastore'
+        return self._use_schema if self._use_schema else 'hive_metastore'
```

Comment:
This is incorrect, catalog is one level above the schema

Reply:
Yes - schema and catalog seem to be used interchangeably sometimes in some documentation - I am used to catalog/db.schema.table. I was going to ask you about that.